teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(name: 'Unknown', age: 0, parent_permission: true, specialization: 'Unknown')
```

Comment:
- [ ] [OPTIONAL] great job setting up the `Teacher` class 👏 it is not actually necessary for `@age` or `@specialization` to have a default value in this case.

Reply:
Thank you for letting me know about these fixable issues! I fixed them step by step. You can check them now. Again, if you spot any missing you are free to mention them! Thank you again!